### PR TITLE
[#IOINFRA-19] Add readme for azure devops

### PR DESCRIPTION
**Pull request by pasqualedevita**

### CODE REVIEW

File: azure-devops/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+1. if your project contains more github repos add all pipelines in the same azure devops project 
+2. for each github repo create a new file with github repo name
+3. put all github repo pipelines into same file `github_repo_name_1.tf`
+4. put all pipelines variables at beginning of `github_repo_name_1.tf`
+
+:warning: **Very Important**
+
+Before apply any changes be sure that permissions on github repo are set as follow:
+1. repo-admins -> Role: admin
```
Comment (balanza) [inline azure-devops/README.md:74]: Why is this needed? I mean, this group can change over time, which is the user that's needed to be admin?

Comment (pasqualedevita) [reply azure-devops/README.md:74]: We can change with pagopa-bot user to admin role. This user need:
1. write webhooks on github repo for PR code review trigger
2. push to master for release tasks

Comment (pasqualedevita) [reply azure-devops/README.md:74]: up! :)

Comment (pasqualedevita) [reply azure-devops/README.md]: @balanza that's all for u? can we merge?

File: azure-devops/README.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+## Requirements
+
+### 1. terraform
+
+In order to manage the suitable version of terraform it is strongly recommended to install the following tool:
+
+- [tfenv](https://github.com/tfutils/tfenv): **Terraform** version manager inspired by rbenv.
+
+Once these tools have been installed, install the terraform version version shown in:
```
Comment (uolter) [inline azure-devops/README.md:13]:
```suggestion
Once these tools have been installed, install the terraform version shown in:
```

File: azure-devops/README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+After installation install terraform:
+
+```sh
+tfenv install
+```
+
+### 2. Azure CLI
+
+In order to authenticate to Azure Infrastructure and manage terraform state it's necessary to install and login to Azure subscription.
```
Comment (uolter) [inline azure-devops/README.md:25]:
```suggestion
In order to authenticate to Azure portal and manage terraform state it's necessary to install and login to Azure subscription.
```

**Review by uolter — CHANGES_REQUESTED** [review]

**Review by uolter — APPROVED** [review]

File: azure-devops/README.md (hunk 1 above)
Comment (balanza) [reply azure-devops/README.md]: it's ok to me
